src/config/urls.ts

```diff
@@ -1,4 +1,12 @@
+export const ENV_DEVELOPMENT = 'DEVELOPMENT'
+export const ENV_PRODUCTION = 'PRODUCTION'
+export const ENV_STAGING = 'STAGING'
```

Comment:
Let's not use predefined environments. Let's do a discovery based on available variables. So at the startup let's go through `process.env` and find all the prefixes. That should produce a list that for the time being will be `['production', 'staging']`